src/components/Quiz/Quiz.css

```diff
@@ -1,3 +1,85 @@
+.quiz-form {
```

Comment:
Good use of responsive-friendly measurements here (vh, rem, flex).  This should make it much easier to resize for different viewers/screen sizes.